engine/classes/ElggVolatileMetadataCache.php

```diff
@@ -289,7 +289,7 @@ public function populateFromEntities($guids) {
 				"JOIN {$db_prefix}metastrings n ON n_table.name_id = n.id",
 			),
 			'selects' => array('n.string AS name', 'v.string AS value'),
-			'order_by' => 'n_table.entity_guid, n_table.time_created ASC',
+			'order_by' => 'n_table.entity_guid, n_table.id ASC',
```

Comment:
why not `n_table.time_created ASC, n_table.id ASC`? If you antidate the metadata this change could provide unexpected results